test/qmi/codec/network_access_test.exs

```diff
@@ -60,7 +60,8 @@ defmodule QMI.Codec.NetworkAccessTest do
                 serving_system_ps_attach_state: :attached,
                 serving_system_radio_interfaces: [:umts],
                 serving_system_registration_state: :registered,
-                serving_system_selected_network: :network_3gpp
+                serving_system_selected_network: :network_3gpp,
+                cell_id: 27_663_611
```

Comment:
Where did you find this cell ID? I'm trying to look it up and I'm getting a place in Milwaukee???

Reply:
I have resolved this due to our testing earlier today, let me know if you still have feedback.